docs/book/developer-guide/materializer.md

```diff
@@ -77,7 +77,7 @@ Each materializer has `ASSOCIATED_TYPES` and `ASSOCIATED_ARTIFACT_TYPES`.
   etc. This is simply a tag to query certain artifact types in the
   post-execution workflow.
 
-### Extending the `BaseMaterializer`
+# Writing a custom materializer
```

Comment:
IMO only the main headline should be h1 and I would use h2 for all subheaders within a page like this one. Might reformatting this whole file tho.